spec/build_spec.lua

```diff
@@ -132,21 +129,21 @@ describe("LuaRocks build tests #integration", function()
    end)
 
    describe("LuaRocks build - basic builds", function()
-      it("LuaRocks build luadoc", function()
+      pending("LuaRocks build luadoc", function()
          assert.is_true(run.luarocks_bool("build luadoc"))
       end)
       
-      it("LuaRocks build luacov diff version", function()
```

Comment:
This one (or preferrably a version of it using fixtures instead of a real module) should probably stay in, as it tests receiving an explicit version number as a second argument (and the choice of luacov was to test building a package that was already installed (ugly, I know — very "implicit" :\ ) )